pages/linux/archlinux-java.md

```diff
@@ -0,0 +1,19 @@
+# archlinux-java
+
+> A helper script that provides functionalities for Java environments.
```

Comment:
```suggestion
> A helper script that provides functionalities for Java environments.
> More information: <https://github.com/michaellass/archlinux-java-run>.
```

Reply:
Thanks for the GitHub repo, @navarroaxel! However, I checked that repo out and I don't think that its the same one that I am writing about, plus, I am unable to find the credit for this script in the ArchWiki. Please correct me if I am wrong. I would love to give credit to the person who made this script.

Reply:
https://aur.archlinux.org/packages/archlinux-java-run The AUR package compiles that repo in the `PKGBUILD`, are you making reference to another package?

Reply:
You are right, my bad, thanks for the clarification, I'll add it in :+1: